config/model-unet.toml

```diff
@@ -29,3 +29,6 @@
 
   # Weight decay l2 penalty for the optimizer
   decay      = 0.0001
+
+  # Loss function name (e.g 'Lovasz', 'mIoU' or 'CrossEntropy')
+  loss = 'Lovasz'
```

Comment:
Thank you, this will be a great default for folks! :bow:

Reply:
:)